src/components/info/templates/default-template.vue

```diff
@@ -0,0 +1,152 @@
+        {{ t(`${layers.layerLabel}`) }}
+      </h1>
+      <div
+        v-for="feature in layers.features"
+        :key="feature.id"
+        class="lux-poi-feature"
+      >
+        <h4 v-if="feature.attributes.label">
+          {{ t(`${feature.attributes.label}`) }}
+        </h4>
+        <div data-cy="defaultTemplateAttributes" v-if="hasAttributes(feature)">
+          <div
+            v-for="entry in prefixKeys(feature.attributes, 'f_').sort(
+              (a, b) => {
+                return layers.ordered ? 0 : a.key.localeCompare(b.key)
+              }
+            )"
+            :key="entry.key"
+          >
+            <span
+              v-if="
+                !isEmpty(entry['value']) &&
+                showAttributesByLang(entry, layers.layer, i18next.language)
+              "
+            >
+              <label v-if="!isLink(entry['value'])"
+                >{{ t(`${entry['key']}`) }} :
+              </label>
+              <span
+                v-if="!isLink(entry['value'])"
+                v-dompurify-html="entry['value']"
+              ></span>
+              <a
+                v-if="
+                  !(
+                    layers.layerLabel.startsWith(
+                      'myenergy_solarkataster_luxemburg'
+                    ) || layers.layerLabel.startsWith('eau_new_Wasserstand')
+                  ) && isLink(entry['value'])
+                "
+                :href="`${entry['value']}`"
+                target="_blank"
+                >{{ t(`${entry['key']}`) }}</a
+              >
+              <a
+                data-cy="defaultTemplateSolarLink"
+                v-if="
+                  layers.layerLabel.startsWith(
+                    'myenergy_solarkataster_luxemburg'
+                  ) &&
+                  isLink(entry['value']) &&
+                  entry['key'] == 'f_href'
+                "
+                :href="`${entry['value']}`"
+                target="_blank"
+              >
+                <button class="lux-solarkataster-button">
+                  <span>{{
+                    t("Lien direct vers le calculateur d'efficacité économique")
+                  }}</span>
+                </button></a
+              >
+              <span
+                v-if="isLink(entry['value']) && entry['key'] == 'f_AudioURL'"
+              >
+                <audio controls autoplay style="width: 260px; height: 50px">
+                  <source :src="`${entry['value']}`" type="audio/wav" />
+                </audio>
+              </span>
+              <iframe
+                width="260px"
+                height="560px"
+                v-if="
+                  layers.layerLabel.startsWith('eau_new_Wasserstand') &&
+                  isLink(entry['value']) &&
+                  entry['key'] == 'f_Graph'
+                "
+                :src="`${getTrustedUrl(entry['value'])}`"
+                title="water level graph"
+              ></iframe>
```

Comment:
Can we have somewhere (in the doc, or in code comment) the corresponding layers to test each cases please?

Reply:
I've added it as a comment to the cypress test file, that uses the same layers in its tests.